nibabel/freesurfer/mghformat.py

```diff
@@ -459,9 +459,13 @@ class MGHImage(SpatialImage):
     """ Class for MGH format image
     """
     header_class = MGHHeader
-    files_types = (('image', '.mgh'),)
+    files_types = (('image', '.mgh'),
+                   ('image', '.mgz'))
```

Comment:
Is this second line necessary?  It may cause problems with the `make_file_map` function.